src/backend/launcher.ts

```diff
@@ -864,11 +860,21 @@ async function runWineCommand({
   return new Promise<{ stderr: string; stdout: string }>((res) => {
     const wrappers = options?.wrappers || []
     let bin = ''
+    const umuSupported = isUmuSupported(wineVersion.type)
+
     if (wrappers.length) {
       bin = wrappers.shift()!
-      commandParts.unshift(...wrappers, wineBin)
+      if (umuSupported) {
+        const umuBin = join(runtimePath, 'umu', 'umu-run')
```

Comment:
Alternatively, you can execute umu_run.py instead, which is more direct and what Lutris does